examples/csi_driver_install_vars.yaml

```diff
@@ -3,16 +3,19 @@ service_instance_id: "bc895dc9-f626-41c0-bfd3-94a0d8942a5f"  #IBM Cloud PowerVS
 region: "mon"                                                #IBM Cloud PowerVS service instance region
 zone: "mon01"                                                #IBM Cloud PowerVS service instance zone
 
+csi_driver_type: "stable"                                    #CSI driver type.
```

Comment:
It will be good to add a comment how these values will be used to fetch the content here and in the README.md as well.